### PR TITLE
menu & shadow description and more

### PR DESCRIPTION
**Pull request by hadasfa**
Documentation with the new format for menu component

### CODE REVIEW

File: src/components/Menu/Menu/__stories__/Menu.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+### Menu with keybord navigation without focus
+<Canvas>
+  <Story name="Menu with keybord navigation without focus">
+    {menuKeyboardNavigationWithoutFocusTemplate.bind({})}
+  </Story>
+</Canvas>
+
+
+## Related components
+<RelatedComponents componentsNames={[]} />
```
Comment (orrgottlieb) [inline src/components/Menu/Menu/__stories__/Menu.stories.mdx]: Add menu related components @hadasfa

File: src/components/Menu/MenuDivider/__stories__/MenuDivider.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+<Canvas>
+  <Story name="Overview">
+    { menuDividerTemplate.bind({}) }
+  </Story>
+</Canvas>
+
+## Props
+<ArgsTable of={ MenuDivider } />
+
+## Variants
```
Comment (orrgottlieb) [inline src/components/Menu/MenuDivider/__stories__/MenuDivider.stories.mdx:37]: remove

File: src/components/Menu/MenuDivider/__stories__/MenuDivider.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+## Use cases and examples
```
Comment (orrgottlieb) [inline src/components/Menu/MenuDivider/__stories__/MenuDivider.stories.mdx:39]: remove

**Review by orrgottlieb — APPROVED** [review]

File: src/components/Menu/Menu/__stories__/Menu.stories.mdx (hunk 1 above)
Comment (hadasfa) [reply src/components/Menu/Menu/__stories__/Menu.stories.mdx]: fixed

File: src/components/Menu/MenuDivider/__stories__/MenuDivider.stories.mdx (hunk 2 above)
Comment (hadasfa) [reply src/components/Menu/MenuDivider/__stories__/MenuDivider.stories.mdx]: done

**Pull request merged**